extensions/resource-deployment/src/ui/toolsAndEulaSettingsPage.ts

```diff
@@ -58,14 +57,28 @@ export class ToolsAndEulaPage extends ResourceTypePage {
 				return false; // we return false so that the workflow does not proceed and user gets to either click acceptEulaAndSelect again or cancel
 			}
 
-			if (!this._isDoneButtonEnabled) {
-				this.wizard.wizardObject.message = {
-					text: localize('deploymentDialog.FailedToolsInstallation', "Some tools were still not discovered. Please make sure that they are installed, running and discoverable"),
-					level: azdata.window.MessageLevel.Error
-				};
-				return false;
+			for (let i = 0; i < this._tools.length; i++) {
```

Comment:
@Charles-Gagnon , I can potentially move this to the 'with validation' method if it works correctly.